webauthn-rs-core/build.rs

```diff
@@ -1,22 +1,29 @@
-use openssl::version::{number, version};
+use std::env;
 
 const OPENSSL_DOC: &str = "https://github.com/kanidm/webauthn-rs/blob/master/OpenSSL.md";
 
 fn main() {
-    // LibreSSL reports as OpenSSL v2.
-    #[allow(clippy::unusual_byte_groupings)]
-    if number() < 0x2_00_00_00_0 {
-        println!(
-            r#"
+    println!();
+    if let Ok(v) = env::var("DEP_OPENSSL_VERSION_NUMBER") {
+        let version = u64::from_str_radix(&v, 16).unwrap();
```

Comment:
This should be an expect with an explanation of what went wrong, unwraps lead to spooky outcomes.
```suggestion
            let version = u64::from_str_radix(&v, 16).expect("Failed to parse OpenSSL version in build.rs");
```